docs/BUILD.md

```diff
@@ -75,7 +75,7 @@ The `-SNAPSHOT` suffix is mandatory.
 `OSS_USER`/`OSS_KEY` environment variables or `ossUser`/`ossKey` gradle build properties 
 (the properties have precedence over environment variables). 
 
-See more configuration details of the oss uploading in [oss-publish.gradle](./gradle-scripts/oss-publish.gradle) file.
+See more configuration details of the oss uploading in [oss-publish.gradle](../gradle-scripts/oss-publish.gradle) file.
```

Comment:
that's the reason why i recommend to use absolute path:
`/gradle-scripts/...` 
and this will be correct for all MD files inside the repo

Reply:
8d8a740